src/setup.jl

```diff
@@ -193,18 +200,21 @@ function build_JuliaInterface(sysinfo::Dict{String, String})
     JULIA_LIBS = filter(c -> c != '\'', ldlibs())
 
     jipath = joinpath(@__DIR__, "..", "pkg", "JuliaInterface")
+    gaproot = gaproot_for_building()
     cd(jipath) do
         withenv("CFLAGS" => JULIA_CFLAGS,
                 "LDFLAGS" => JULIA_LDFLAGS * " " * JULIA_LIBS) do
-            run(pipeline(`./configure $(gaproot())`, stdout="build.log"))
+            run(pipeline(`./configure $(gaproot)`, stdout="build.log"))
             run(pipeline(`make V=1 -j$(Sys.CPU_THREADS)`, stdout="build.log", append=true))
         end
     end
 
+    sysinfo = read_sysinfo_gap(joinpath(gaproot, "sysinfo.gap"))
+
     return normpath(joinpath(jipath, "bin", sysinfo["GAParch"]))
```

Comment:
If we add the global `sysinfo` back, then this could also be
```suggestion
    return normpath(joinpath(jipath, "bin", GAP.sysinfo["GAParch"]))
```
assuming that something like `import ...GAP: GAP` is added at the top of this file